index.html

```diff
@@ -25,7 +25,7 @@
             <img src="./src/assets/world.png" alt="world icon" />
           </div>
           <div class="content-select">
-            <select>
+            <select class="arrow">
```

Comment:
what do you think about "custom-arrow" or "arrow-bottom" or similar?

Reply:
I chose `arrow-bottom`. thanks